cromwell-backend/src/main/scala/cromwell/backend/Backend.scala

```diff
@@ -0,0 +1,39 @@
+package cromwell.backend
+
+import cromwell.backend.model.Subscription
+import cromwell.caching.Cacheable
+
+/**
+  * Defines basic functionality to interact with a Backend.
+  */
+trait Backend extends Cacheable {
```

Comment:
One thought here (warning, I'm just quickly scanning - not thinking in depth). All of these functions are Unit=>Unit which normally would be a huge red flag to me. I'd make it clear that these are just tied to the corresponding messages in the backend actor - at that point it's fine since sadly Akka is Any=>Unit anyways. (Oh if only typed actors were around when we started ...)